### ISSUE TITLE
import_nsrl_hashes.py fails with unicode string parsing error

### ISSUE THREAD

**Issue by atkinsj** [google/grr#641]
Hey folks,

This command fails:
`/usr/share/grr-server/bin/python2 /usr/src/grr/grr/server/grr_response_server/bin/import_nsrl_hashes.py --config /usr/share/grr-server/install_data/etc/grr-server.yaml --filename NSRLFile.txt`

with the below message:

```
Traceback (most recent call last):
  File "/usr/src/grr/grr/server/grr_response_server/bin/import_nsrl_hashes.py", line 104, in <module>
    flags.StartMain(main)
  File "/usr/share/grr-server/local/lib/python2.7/site-packages/grr_response_core/lib/flags.py", line 146, in StartMain
    main(sys.argv)
  File "/usr/src/grr/grr/server/grr_response_server/bin/import_nsrl_hashes.py", line 98, in main
    imported = ImportFile(store, filename, flags.FLAGS.start)
  File "/usr/src/grr/grr/server/grr_response_server/bin/import_nsrl_hashes.py", line 40, in ImportFile
    reader = csv.reader(fp, delimiter=",", quotechar="\"")
TypeError: "delimiter" must be string, not unicode
```

Patching the line as follows does resolve the issue:
`reader = csv.reader(fp, delimiter=str(u','), quotechar=str(u'"'))`

This may not be the best approach and I know you're in the midst of some python2/3 porting, so met know what the preferred approach would be and I'm happy to throw a quick PR.

**Comment by mbushkov** [comment]
Thanks for the report! While we're at it, I've asked about NSRL a few weeks ago in GRR Users group (https://groups.google.com/forum/#!search/nsrl$20grr/grr-users/C45ex3y0TDg/lJfiXV4BCQAJ). We're refactoring our file store implementation and are wondering about the NSRL functionality (to be more precise, we have a certain inclination to drop it). Why and how is current NSRL support useful for you?

**Comment by atkinsj** [comment]
I don't anticipate it being valuable at all -- I'm only flicking it so I can try to quantify that assumption. ;)

**Comment by mrhania** [comment]
Fixed in e12f350d4d524 with slightly more sophisticated solution that should ensure compatibility both with Python 2 and Python 3.
